keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -262,7 +262,7 @@ Consider including folks who also work outside the SIG or subproject.
 ## Design Details
 
 A new out-of-tree flag named `PreventVolumeModeConversion` will be introduced on 
-`external-snapshotter` and `external-provisioner`. Both of these components are 
+`snapshot-controller` and `external-provisioner`. Both of these components are 
```

Comment:
If you change "external-snapshotter" to "snapshot-controller" which is the binary name, please also change "external-provisioner" to "csi-provisioner" accordingly.

Reply:
Done! This was to address a comment on the previous update to this KEP